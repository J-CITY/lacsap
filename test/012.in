{$ qwer}
program HelloWorld;{
 $QWER
}
{
   $ QWERTY

}
var
    _int, lazy_town_ : integer;
    __r : real;
Begin
  _int := 1;
END.
