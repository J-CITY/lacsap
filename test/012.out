{$ qwer} 0 0 directive 8004
program 1 0 keyword 334
HelloWorld 1 8 identifier 4000
; 1 18 separator 4
{$QWER} 1 19 directive 8004
{$ QWERTY } 4 0 directive 8004
var 8 0 keyword 316
_int 9 4 identifier 4001
, 9 8 separator 3
lazy_town_ 9 10 identifier 4002
: 9 21 separator 5
integer 9 23 identifier 4003
; 9 30 separator 4
__r 10 4 identifier 4004
: 10 8 separator 5
real 10 10 identifier 4005
; 10 14 separator 4
Begin 11 0 keyword 324
_int 12 2 identifier 4001
:= 12 7 operation 23
1 12 10 literalInt 7000
; 12 11 separator 4
END 13 0 keyword 315
. 13 3 separator 6
